ArduPlane/commands.cpp

```diff
@@ -136,17 +136,11 @@ void Plane::update_home()
         return;
     }
     if (home_is_set == HOME_SET_NOT_LOCKED) {
-        Location loc = gps.location();
-        Location origin;
-        // if an EKF origin is available then we leave home equal to
-        // the height of that origin. This ensures that our relative
-        // height calculations are using the same origin
-        if (ahrs.get_origin(origin)) {
-            loc.alt = origin.alt;
-        }
+        Location loc;
+        ahrs.get_position(loc);
         ahrs.set_home(loc);
```

Comment:
Setting home should only be done if we got a position, will push a fix in a bit.